src/components/meta.svelte

```diff
@@ -20,4 +20,6 @@
 	<meta property="twitter:url" content="https://{url}" />
 	<meta name="twitter:title" content={title} />
 	<meta name="twitter:description" content={description} />
+
+	<script defer data-domain="svelte-custom-renderers.com" src="https://plausible.io/js/script.js"></script>
```

Comment:
Given this is a static external script that isn't touched by Vite in any way, I think it would be more appropriate to put it straight into `src/app.html`